Marlin/example_configurations/delta/FLSUN/auto_calibrate/Configuration.h

```diff
@@ -728,7 +728,7 @@
   #define Z_PROBE_ALLEN_KEY_DEPLOY_2_X 0.0
   #define Z_PROBE_ALLEN_KEY_DEPLOY_2_Y DELTA_PRINTABLE_RADIUS
   #define Z_PROBE_ALLEN_KEY_DEPLOY_2_Z 100.0
-  #define Z_PROBE_ALLEN_KEY_DEPLOY_2_FEEDRATE (XY_PROBE_SPEED)/10
+  #define Z_PROBE_ALLEN_KEY_DEPLOY_2_FEEDRATE (XY_PROBE_SPEED/10)
```

Comment:
Wrong. The parentheses belong around `XY_PROBE_SPEED` to make it a unit for division. (Set `XY_PROBE_SPEED` to "300 + 100" with no parentheses and you'll see why this is done.)